### ISSUE TITLE
Apple complains about show: hide: setLabelText:

### ISSUE THREAD

**Issue by jeabraham** [matej/MBProgressHUD#70]
Apple recently said:

     We have discovered one or more issues with your recent delivery. To process your delivery, the following issues must be corrected:
     Non-public API usage:
     The app references non-public selectors: hide:, setLabelText:, show:

Anyone else run into this? Does it make any sense to rename these methods to avoid Apple's warning?

**Comment by matej** [comment]
What? How do you know it's `MBProgressHUD`? That's all public API.

**Comment by jeabraham** [comment]
I searched all my code for those symbol names and only found them in MBProgressHUD. I've been googling and I guess it's just an automated warning from Apple, supposedly the app will still be accepted. Apple seems to think that naming methods "hide" and "show" is somehow related to private API use. I guess the only tool they have to automatically check for the use of private APIs is a symbol name checker.

I'll let you know if it amounts to anything.

--
John Abraham

**Comment by jeabraham** [comment]
It might be that I submitted a build without including the MBProgressHUD Framework.  I submitted a new build, checking the build settings to be sure to include all frameworks, and it got approved right away, without the warning.

Perhaps it didn't link correctly, and Apple thought I was trying to call hide() and show() in the system OS.

Anyway I'll let you know if the warning comes up again.

--
John Abraham
jea@hbaspecto.com
403-232-1060
